### PR TITLE
Build on fedora, fix favicons, add some extensions

### PR DESCRIPTION
**Pull request by razzeee**
- Docs website is now build on fedora, but still hosted via ubuntu (allows for newer versions)
- Updated the `website` icons for more cases - some of these might be considered unnecessary
- Add opengraph plugin and some metadata for nicer social cards
- Added a copy button to code boxes, I'm not 100% sure how relavant it is in these docs, but it's nice to have
- Add a favicon for the docs website

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -15,46 +15,41 @@ jobs:
...
+            geoclue2-devel \
+            glib2-devel \
+            json-glib-devel \
+            pipewire-devel \
+            libportal-devel \
+            systemd-devel \
             llvm \
             meson \
-            python3-gi \
+            python3-gobject \
```
Comment (GeorgesStavracas) [inline .github/workflows/pages.yml:43]: Please sort this list alphabetically (don't ignore the `lib`  prefix :slightly_smiling_face:)

File: .github/workflows/pages.yml [hunk 2]
```diff
@@ -15,46 +15,41 @@ jobs:
...
-          DEBIAN_FRONTEND: noninteractive
         run: |
-          apt-get install -y --no-install-recommends \
+          dnf install -y \
             ca-certificates \
-            furo \
             git \
-            python3-sphinx
+            python3-sphinx \
+            python3-furo
```
Comment (GeorgesStavracas) [inline .github/workflows/pages.yml]: Ditto

File: doc/website/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff"
```
Comment (GeorgesStavracas) [inline doc/website/site.webmanifest:17]: Is this correct, actually?

File: .github/workflows/pages.yml [hunk 4]
```diff
@@ -50,7 +50,8 @@ jobs:
             git \
             python3-sphinx \
             python3-furo \
-            python-sphinxext-opengraph
+            python-sphinxext-opengraph \
+            python-sphinx-copybutton
```
Comment (GeorgesStavracas) [inline .github/workflows/pages.yml]: Ditto about alphabetical order

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]

File: .github/workflows/pages.yml (hunk 1 above)
Comment (razzeee) [reply .github/workflows/pages.yml]: Didn't even cross my mind, that it was alphabetical :)

File: doc/website/site.webmanifest (hunk 3 above)
Comment (razzeee) [reply doc/website/site.webmanifest:17]: Probably not, but it also probably doesn't matter. The Theme color might.

Still, tried to see how this looks and currently, it doesn't even find the images above.

Comment (razzeee) [reply doc/website/site.webmanifest]: Should be fixed. Please let me know, if I should rebase and squash these to the relevant commit.

image

**Review by GeorgesStavracas — APPROVED** [review]

**Pull request merged**